src/Tes.SDK.Tests/TesClientTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json;
+using Moq;
+using Tes.Models;
+
+namespace Tes.SDK.Tests
+{
+    [TestClass]
+    public class TesClientTests
+    {
+        private ITesClient _client;
```

Comment:
`private ITesClient _client = null!;` will quiet the warnings. ditto for next line